site/docs/education/egeria-dojo/developer/develop-asset-listen-cut-paste.md

```diff
@@ -0,0 +1,34 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+??? beginner "Create a new IntelliJ project for AssetListen called `egeria-dojo1`"
+    ---8<-- "docs/education/tutorials/intellij-tutorial/intellij-new-project.md"
+
+??? beginner "Create a new *Sources Root* in `egeria-dojo1` under `src` called `main/java`"
```

Comment:
Normally sources root gets set automatically when a maven pom (or gradle build file) is imported. However in this sequence you are creating package/class before the pom (quite reasonably) so it's needed sooner. 

One alternative is to create the pom first. Just mentioning for considerations as it removes steps, these instructions are still fine

Reply:
The aim is to help people understand how intellij works